app/controllers/patron_requests_controller.rb

```diff
@@ -39,6 +39,9 @@ def current_request
 
   def assign_new_attributes
     @patron_request.assign_attributes(**new_params)
+    # This validation prevents an injection attack in GraphQL. Ideally, this would be a model validation,
+    # but there are other validations that require first running a GraphQL query.
+    raise ActionController::BadRequest unless /\Aa?\d+\z/.match?(new_params[:instance_hrid])
```

Comment:
This regex is.. starts with an `a` or a digit? We also have hrids that start with `L`, or `in`, and probably arbitrary now in FOLIO.

Reply:
Yes. I've updated it include those cases as well.

Reply:
How about `\w+`?  It seems bad for people to be injecting quotes, etc.